File: Retos/Reto #12 - VIERNES 13 [Fácil]/python/ycanas.py
import datetime

def has_friday_thirteenth(month: int, year: int) -> bool:
    FRYDAY = 4
    DAY = 13

    date = datetime.date(year, month,  DAY)
    result = date.weekday() == FRYDAY
    return result


print(has_friday_thirteenth(3, 2023))
print(has_friday_thirteenth(10, 2023))

File: Retos/Reto #12 - VIERNES 13 [Fácil]/java/ycanas.java
import java.time.LocalDate;
import java.time.DayOfWeek;

public class ycanas{
    public static Boolean hasFridayThirteenth(int month, int year){
        final int DAY = 13;

        LocalDate date = LocalDate.of(year, month, DAY);
        DayOfWeek day  = date.getDayOfWeek();

        return day == DayOfWeek.FRIDAY;
    }

    public static void main(String[] args){
        System.out.println(hasFridayThirteenth(10, 2023));
        System.out.println(hasFridayThirteenth(3, 2023));
    }
}